src/main/java/Task.java

```diff
@@ -0,0 +1,74 @@
+import java.util.ArrayList;
+
+public class Task {
+    
+    /*
+     ArrayList of all tasks
+     All subclasses of Task call the parent constructor, which adds them to the list
+    */
+    public static ArrayList<Task> tasks = new ArrayList<Task>(); 
+
+    // Task class attributes
+    String description;
+    boolean isComplete;
```

Comment:
I think you can use private/protected for the attribute for better data encapsulation